packages/dx-react-chart-material-ui/src/templates/legend/item.test.jsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { createShallow, getClasses } from 'material-ui/test-utils';
+import { ListItem } from 'material-ui/List';
+import { Item } from './item';
+
+describe('Root', () => {
+  const shallow = createShallow({ dive: true });
+  const classes = getClasses(<Item>children</Item>);
```

Comment:
Usually, we create `shallow` and `classes` inside the `beforeAll` function. Material UI [uses](https://github.com/mui-org/material-ui/blob/v1-beta/packages/material-ui/src/Checkbox/Checkbox.test.js#L13) the similar approach. I'm not sure if it's strongly required. Just find this difference.